dataland-frontend/src/components/forms/CreateEUTaxonomyForNonFinancials.vue

```diff
@@ -331,6 +331,9 @@
                                 name="page"
                                 placeholder="Page"
                                 validation-label="Page"
+                                validation="min:0"
+                                step="1"
+                                min="0"
                               />
                             </div>
```

Comment:
Question. Currently "Revenue" is limited to not negative values. Should this be changed?

Reply:
Ok, then i resolve this conversation.